src/styles/Utills/Normalize.scss

```diff
@@ -0,0 +1,7 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
```

Comment:
If you you use the proprty you don't need to apply it for every element in your project like you did it above